ingest/usecase/plugins/orderbook/fillbot/osmosis_swap.go

```diff
@@ -324,36 +323,18 @@ func (o *orderbookFillerIngestPlugin) simulateMsgs(ctx context.Context, msgs []s
 	txFactory = txFactory.WithGasAdjustment(1.02)
 
 	// Estimate transaction
-	gasResult, adjustedGasUsed, err := CalculateGas(ctx, o.passthroughGRPCClient.GetChainGRPCClient(), txFactory, msgs...)
+	gasResult, adjustedGasUsed, err := tx.CalculateGas(
+		o.passthroughGRPCClient.GetChainGRPCClient(),
+		txFactory,
+		msgs...,
+	)
 	if err != nil {
 		return nil, adjustedGasUsed, err
 	}
 
 	return gasResult, adjustedGasUsed, nil
 }
 
-// CalculateGas simulates the execution of a transaction and returns the
-// simulation response obtained by the query and the adjusted gas amount.
-func CalculateGas(
```

Comment:
This was duplicate of [tx.CalculateGas](https://github.com/cosmos/cosmos-sdk/blob/ee3d320eaa55e5aab1bf216f03a8d8aa5833d549/client/tx/tx.go#L148)